src/hexdoc/_templates/category.html.jinja

```diff
@@ -1,6 +1,8 @@
 {% import "macros/formatting.html.jinja" as fmt with context %}
 
-<section id="{{ category.id.path }}">
+<section id="{{ category.id.path }}" class="{% if category.flag is not none %}
+    {{ category.flag.css_classnames() }}
+  {% endif %}">
```

Comment:
```suggestion
<section id="{{ category.id.path }}" class="{{ category.flag.css_classnames() if category.flag }}">
```

Reply:
(this is also helpful because the current code adds extra newlines to the output, as you can see in the snapshots)